jinja_utils_test.py

```diff
@@ -108,3 +108,16 @@ def test_evaluate_object(self):
         parsed_dict = jinja_utils.evaluate_object(orig_dict, {'b': 'c'})
         self.assertEqual(orig_dict, {'a': '{{b}}'})
         self.assertEqual(parsed_dict, {'a': 'c'})
+
+    def test_interpolate_cache_slug(self):
+        parsed_str = jinja_utils.parse_string(
```

Comment:
This is testing parse_string(), not interpolate_cache_slug(). You want to test the latter.

Try also testing it with strings like 'abc{{cache_slug}}' and '{{invalid_variable}}'